### PR TITLE
Adding ECS Container Services

### PR DESCRIPTION
**Pull request by aimanafzal**
Please review my PR @ChristianWitts

### CODE REVIEW

File: providers/aws/ecs/containers.go [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/ecs"
+	"github.com/aws/aws-sdk-go-v2/service/sts"
+	. "github.com/tailwarden/komiser/models"
+	. "github.com/tailwarden/komiser/providers"
+)
+
+func Container(ctx context.Context, client ProviderClient) ([]Resource, error) {
+	resources := make([]Resource, 0)
```
Comment (mlabouardy) [inline providers/aws/ecs/containers.go:18]: `ContainerInstances` just to follow the naming convention :)

File: providers/aws/aws.go [hunk 2]
```diff
@@ -46,6 +46,7 @@ func listOfSupportedServices() []providers.FetchDataFunction {
 		ecs.Tasks,
 		ecs.Services,
 		ecs.Clusters,
+		ecs.Container,
```
Comment (mlabouardy) [inline providers/aws/aws.go:49]: `ecs.ContainerInstances`

**Review by mlabouardy — CHANGES_REQUESTED** [review]
Thanks @aimanafzal
I requested a small change :)

File: providers/aws/aws.go (hunk 2 above)
Comment (aimanafzal) [reply providers/aws/aws.go]: Done :)

File: providers/aws/ecs/containers.go (hunk 1 above)
Comment (aimanafzal) [reply providers/aws/ecs/containers.go]: Done :)

**Review by mlabouardy — APPROVED** [review]

**Pull request merged**